news/pymbar_bootstrap.rst

```diff
@@ -0,0 +1,23 @@
+**Added:**
+
+* <news item>
+
+**Changed:**
+
+* The MBAR bootstrap (1000 iterations) error is used to estimate protocol uncertainty and pymbar3 is no longer supported `PR#1077 <https://github.com/OpenFreeEnergy/openfe/pull/1077>`_
```

Comment:
@jthorton could you add an "instead of" here too maybe? (i.e. tell them what we used to do for MBAR estimation).

Reply:
Done in [44f1b28](https://github.com/OpenFreeEnergy/openfe/pull/1077/commits/44f1b283fb92385549dc26587c5212ba6bed94ac)